core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedClusterMessage.scala

```diff
@@ -132,4 +132,6 @@ private[spark] object CoarseGrainedClusterMessages {
   // Used internally by executors to shut themselves down.
   case object Shutdown extends CoarseGrainedClusterMessage
 
+  // The message to check if scheduler thinks the executor is alive or not.
```

Comment:
can we be more specific about the "scheduler" here? which scheduler?

Reply:
updated to `CoarseGrainedSchedulerBackend`